scripts/get_cmake_vars/CMakeLists.txt

```
@@ -92,12 +92,16 @@ endforeach()
 macro(_vcpkg_adjust_flags flag_var)
     if(MSVC) # Transform MSVC /flags to -flags due to bash scripts intepreting /flag as a path.
         string(REGEX REPLACE "(^| )/" "\\1-" ${flag_var} "${${flag_var}}")
+        if(CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
+            foreach("${flag_var}" IN_LIST CMAKE_C_FLAGS CMAKE_CXX_FLAGS)

```

```suggestion
            foreach("${flag_var}" IN_LIST CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
```

this is a very, very weird line; what this will do is set `${flag_var}` to first `IN_LIST` (and append "-ZW:nostdlib" to `IN_LIST`), then the same for CMAKE_C_FLAGS and CMAKE_CXX_FLAGS; did you mean to do something like

```suggestion
            foreach(var IN ITEMS CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
```
?

---

Good catch. It should simply be a `if()` block not a `foreach()` block. Maybe a merge messed it up.